sig-node/annual-report-2022.md

```diff
@@ -44,40 +51,48 @@
 1. What areas and/or subprojects does your group need the most help with?
    Any areas with 2 or fewer OWNERs? (link to more details)
 
-   -
-   -
-   -
+   - Node Problem Detector is not very active and has small participation.
+   - In general all areas are well covered with reviewers and approvers. However community would benefit to have clearer ownership of areas defined someplace. We plan to:
+     - Add more people to subdirectories as approvers
+     - Split more clearly between approvers and reviewers
 
 2. What metrics/community health stats does your group care about and/or measure?
 
-   -
-   -
-   -
+   - Active PRs and weekly changes of PRs.
+   - Untriaged PRs, bugs
```

Comment:
It would be good to pay attention to the full PR cycle time, especially to a review and approval parts.